ckanext/datastore/config_declaration.yaml

```diff
@@ -92,6 +92,16 @@ groups:
       them. It can be overwritten by the user by passing the "lang" parameter to
       "datastore_search" and "datastore_create".
 
+  - default: 60_000
```

Comment:
let's set the default to a much smaller value (like 1s) because on busy sites one request being able to fully load the db server for 60s is really bad.

Reply:
Done here https://github.com/avdata99/ckan/commit/2d235309d02f363c6932f5493594ff17aafffd01